### PR TITLE
Check in dirty examples of new rr2

### PR DESCRIPTION
**Pull request by chenglou**
They don't compile here yet, but they work otherwise. Submitting this for the adventurous folks. Don't merge.

### CODE REVIEW

File: examples/simple/index.nobuild.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    window.propsReceivedIcon = "🠫";
+    window.updateIcon = "♼";
+    window.elbowIcon = "⮡";
+  </script>
+</head>
+<body style="">
+  <div id="index"></div>
+  <!-- <script src="../../bundledOutputs/simple.js"></script> -->
+  <script src="./console.style.js"></script>
+  <script data-main="../../lib/js/src/simple/simpleRoot.js" data-project-root="../../" src="../require_polyfill.js">
```
Comment (jordwalke) [inline examples/simple/index.nobuild.html:19]: Thank you for keeping the nobuild versions!

Comment (chenglou) [reply examples/simple/index.nobuild.html:19]: Oh I just copied things over lol.

I might move these arounds though. Otherwise it might be confusing.

Comment (jordwalke) [reply examples/simple/index.nobuild.html:19]: In any event, I really like having a webpack-less require-polyfill based workflow even if webpack is used for production mode.

This is what I was telling @knowbody

Comment (knowbody) [reply examples/simple/index.nobuild.html:19]: It's coming soon! I got something working just cleaning up, will send the PR to CRRA probably today.

**Comment by chenglou** [comment]
A bit stale. Closing

**Pull request closed without merging**